src/wazuh_modules/wm_office365.h

```diff
@@ -19,16 +19,24 @@
 #define WM_OFFICE365_DEFAULT_INTERVAL 60
 #define WM_OFFICE365_DEFAULT_CURL_MAX_SIZE 1048576L
 #define WM_OFFICE365_DEFAULT_CURL_REQUEST_TIMEOUT 60L
+#define WM_OFFICE365_DEFAULT_API_LOGIN_FQDN "login.microsoftonline.com"
+#define WM_OFFICE365_DEFAULT_API_MANAGEMENT_FQDN "manage.office.com"
+
+#define WM_OFFICE365_GCC_API_LOGIN_FQDN "login.microsoftonline.com"
+#define WM_OFFICE365_GCC_API_MANAGEMENT_FQDN "manage-gcc.office.com"
+
+#define WM_OFFICE365_GGC_HIGH_API_LOGIN_FQDN "login.microsoftonline.us"
```

Comment:
Definitely remember fixing this typo. Must have forgotten to push the change from the dev system when I did all the unit testing.

Reply:
Done!